test/test_value_converter_factory.rb

```diff
@@ -268,6 +288,15 @@ def test_timestamp
           assert_equal expected, converter.call(Time.at(subject).utc)
         end
 
+        def test_date
+          converter = ValueConverterFactory.new(SCHEMA_TYPE, 'DATE').create_converter
```

Comment:
You don't have to consider timezone like `timestamp->string` conversion?
https://github.com/trocco-io/embulk-output-bigquery/pull/1/files#diff-083c4f3985a5d49e74d49fddbb714d3bR276

Reply:
It may be useful if I can select timezone I will convert value from timestamp to date with.

for example:
`2019-10-02 00:00:00 +09:00`
when `JST`: `2019-10-02`
when `UTC`: `2019-10-01`

`2019-10-01 15:00:00 +00:00`
when `JST`: `2019-10-02`
when `UTC`: `2019-10-01`

Reply:
addded